essay.tex

```diff
@@ -11,35 +11,71 @@
+\\~\\
+This is what the Agile manifesto says about sprints ``Deliver working software frequently, from a couple of weeks to a couple of months, with a preference to the shorter timescale.``\cite{three}.
+\\~\\
+So Sprints are an important part of the development cycle for games and they need to be well planned and organized in order for the team to work efficiently on a game.
+
+\section{So how long should a sprint last?}
+A lot of sources say that sprints should last 2-4 weeks depending on the team and the scale of the project \cite{two} \cite{four} \cite{five}. Some sources that have been explored also suggest that shorter sprints are generally better for software and games development \cite{three}. Shorter sprints generally work in favour for the Agile Scrum methodology for being short iterative steps towards development. A lot of teams have mentioned that they generally have 2 week long sprints as their main sprint duration as it gives a good time frame for developers to work on the scrum backlog. Any more than 4 weeks is usually considered too long and unfitting for the agile philosophy as it means the developers aren't iterating as much.
+\\~\\
+Some teams like to change up the sprint duration during development because they might be at a different stage that requires more or less time for sprint runs. So for example a team could start off developing a game with 3 week long sprints and then change to 2 week long sprints once they have finished the alpha stage of the game and can now work on implementing more content instead of working on the overall framework.
+
+\section{The idea of 1 week sprints?}
+1 week long sprints are rarely mentioned and are a fairly controversial idea, and for good reasons. This Essay wants to shed some light onto the positive and negatives of the weekly sprint and argue for why it's actually a good idea and that game dev teams should practice it more.
+\\~\\
+So let's start off with the negatives:\\
+Some sources say that 1 week sprints are too short and don't give enough time for teams to work everything they wanted to work on from their sprint planning, other sources say that 1 week sprints aren't good because it means a loss of time from having to do more sprint planning meetings and review meetings.
+\\~\\
+Now let's look at the positives:\\
+1 week or 5 working days is actually quite a long time when you think about it and a lot can be achieved during that time frame.
+An article by Dan Ackerson suggests that developers get sloppy and don't work as productive as productive as they could be in sprints that last more than a week and encourages the idea of 1 weeks sprints as it gets the team more focused on the sprint goals\cite{six}.
```

Comment:
You don't need to explicitly state the nature of the source. Simply put: `Ackerson \cite{ackerson-productivity} suggests that developers...`